include/baresip.h

```diff
@@ -465,6 +465,14 @@ struct contact *contacts_current(const struct contacts *contacts);
 struct mediadev {
 	struct le   le;
 	char  *name;
+	struct {
+		int host_idx;
+		int dev_idx;
+		bool default_src;
+		bool default_play;
+		uint32_t in_channels;
+		uint32_t out_channels;
+	} info;
```

Comment:
This can be split in src/play:

```c
struct {
    uint32_t channels;
    bool is_default;
} src, play;
```

... since the property types for AUSRC and AUPLAY are the same, most of the time ...